math/radian/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        randy3k radian 0.6.4 v
+revision            0
+categories          math python R
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         A XXI century R console
+long_description    radian is an alternative console for the R program with multiline editing and rich syntax highlight. \
+                    One would consider radian as a ipython clone for R, though its design is more aligned to Julia.
+
+checksums           rmd160  b7cc661d678becac17dab113d75366414dfa4784 \
+                    sha256  ccdca2a3c3079dbdea393a0fa081de26344af290c370a6990cd7d3ad4982b2a0 \
+                    size    74412
+supported_archs     noarch
+
+python.default_version 310
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+depends_lib-append  port:py${python.version}-prompt_toolkit \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pytest-runner \
```

Comment:
this should only be a build dependency correct; it's typically not needed at runtime